### PR TITLE
Sp 671 migrate assessment api calls to sp api

### PR DESCRIPTION
**Pull request by jonhindle**

### CODE REVIEW

File: common/middleware/getOffenderDetails.test.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    })
+    it('should return an object with the required values', () => {
+      expect(res.locals.offenderDetails).toEqual({
+        fullName: 'Bernard Shakey',
+        crn: 'S000001',
+        noms: 'A0000AB',
+      })
+    })
+    it('should call the data service once and pass the id', () => {
+      expect(getOffenderData).toHaveBeenCalled()
```
Comment (Coupey) [inline common/middleware/getOffenderDetails.test.js:43]: Update expectation to show it has indeed been called once? 

`expect(getOffenderData).toHaveBeenCalledTimes(1)`

Comment (jonhindle) [reply common/middleware/getOffenderDetails.test.js:43]: I think I could replace all the `toHaveBeenCalled` methods with `toHaveBeenCalledTimes(1)` 🤔 I'll do that 👍

**Comment by Coupey** [comment]
My docker-compose comment above is probably wrong - other services will still need it running even when local.

**Comment by jonhindle** [comment]
So I didn't pull it out of the `docker-compose.yml`, because its listed as a requisite of the sentence planning API, so I thought it needed leaving in? The mock should just still work.

**Pull request merged**